roles/monitoring_sap/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# This playbook checks, if SAP_HANA_CLIENT is insttalled  on the monitoring host 
+- name: Check existence of /usr/lib64/python3.6/site-packages/hdbcli/dbapi.py
```

Comment:
Move the tasks to a block and add condition when: action=create

Reply:
moved to a block controlled by var.